invenio.cfg

```
@@ -201,6 +201,7 @@ THEME_FOOTER_TEMPLATE = 'zenodo_rdm/footer.html'
 # Enable the below on the dedicated environment
 # THEME_TRACKINGCODE_TEMPLATE = "zenodo_rdm/trackingcode/trackingcode-qa.html"
 
+COOKIE_POLICY = "https://about.zenodo.org/cookie-policy"
```

```suggestion
COOKIE_POLICY_URL = "https://about.zenodo.org/cookie-policy"
```

nit: naming wise, I'd make it more explicit that it's a URL. To avoid clashes In case we want to add a flag to enable or disable the banner